omod/src/main/webapp/tag.jsp

```diff
@@ -0,0 +1,21 @@
+<%@ include file="/WEB-INF/template/include.jsp"%>
+
+<%@ include file="/WEB-INF/template/header.jsp"%>
+
+<h2><spring:message code="tag-omod.title" /></h2>
```

Comment:
This is why I hinted that after generating the module, you need to clean out everything you don't need first